tools/driver/CMakeLists.txt

```diff
@@ -13,18 +13,38 @@ add_swift_host_tool(swift
 )
 
 target_link_libraries(swift edit)
+# CMake doesn't support create_symlink on Windows: http://cmake.3232098.n2.nabble.com/cmake-E-create-symlink-for-Windows-td4151271.html
```

Comment:
Can we wait on this? The reason why I ask is that I'm going to submit a PR fixing some symlink issues on Windows, and this code will go.
The problem is that there is no good way to check if a symlink exists on Windows using CMake, and `mklink` reports an error which breaks/stops the build.
The fix would be to create a windows specific (or Xplat python script) that checks if a symlink exists before creating it.